### ISSUE TITLE
Sensors not working; caused by "FROMLIST: Input: atkbd: Receive and use physcode->keycode mapping from FW" (?)

### ISSUE THREAD

**Issue by kitakar5525** [kitakar5525/chromeos-kernel-linux-surface#14]
At least on Surface 3, I found sensors seem to be not working (no auto-rotation with [this patch](https://github.com/sebanc/brunch/commit/d9c46c812046d48772ec31498792dab6a4419b25)) on the latest chromeos-4.19 `4.19.114`.

This issue won't happen on the latest linux-surface 4.19 kernel (`4.19.114-1-surface-lts`) with Arch Linux.
Also not happen on chromeos-5.4 `5.4.30`.

I tried chromeos-4.19 `4.19.113` ([commit 5080748](https://chromium.googlesource.com/chromiumos/third_party/kernel/+/5080748c9c5777fb5e0c977afad7a5bccd841719)) and it's working. After chromeos-4.19 `4.19.114` ([commit 0f61ed8](https://chromium.googlesource.com/chromiumos/third_party/kernel/+/0f61ed8ddae529768a586c5ae7a8b28879048b27)), it's not working anymore.

I haven't yet found what is causing this issue.

**Comment by kitakar5525** [comment]
Hmm. It's not working on branch [release-R83-13020.B-chromeos-4.19](https://chromium.googlesource.com/chromiumos/third_party/kernel/+/refs/heads/release-R83-13020.B-chromeos-4.19), too.

**Comment by kitakar5525** [comment]
Hmm... Reverting [this commit](https://chromium.googlesource.com/chromiumos/third_party/kernel/+/c29d72eb8a0919320de503ed339c07d8b1a8d7a0) fixed this issue. 

Need to figure out why the commit caused this issue...

**Comment by kitakar5525** [comment]
The aforementioned commit is on `v4.19.113.r8526.c29d72eb8a09`

**Comment by kitakar5525** [comment]
Hmm... on previous comments, I tested on non-brunch chromeos. Now, I tried with brunch chromeos and sensors are working with that aforementioned commit not reverted.
I tried even without brunch's userspace patches `99-preui_probe.sh` and `99-delay_ui.sh` but sensors are still working.

Not sure what fixed this issue... but actually fixed. Closing this issue as resolved.

Note for non-brunch users: If you still encounter this issue, try doing the same thing as this brunch patch: 99-preui_probe.sh (https://github.com/sebanc/brunch/blob/r83/patches/99-preui_probe.sh). (Create udev rule under `/etc/init/preui-probe.conf` with [this content](https://github.com/sebanc/brunch/blob/22ab2a36b78c4f311ed3a50eba9555b6e948c3c0/patches/99-preui_probe.sh#L7-L12))
